py3pin/Pinterest.py

```diff
@@ -126,6 +126,7 @@ def login(self, headless=True, wait_time=15, proxy=None):
         chrome_options = Options()
         if headless:
             chrome_options.add_argument("--headless")
+            chrome_options.add_argument("--lang=en")
```

Comment:
Should this be outside of the if? If login is not headles

Reply:
I guess it would make sense in both cases, but yes

Reply:
Both cases